.bingo/README.md

```diff
@@ -0,0 +1,14 @@
+# Project Development Dependencies.
```

Comment:
Bingo changes the go tooling for the entire repo. Can you please move this to a separate PR?

Reply:
I agree. +1 on Bingo and +1 on a separate PR to make that decision as a team.